portal/code/company-portal/src/components/dragdrop.tsx

```diff
@@ -18,16 +19,22 @@ interface DragDropProps {
 export const DragDrop = ({currentActiveStep, addCurrentStep}: DragDropProps) => {
   const { t } = useTranslation();
 
+  useEffect(() => {
+    window.scrollTo(0, 0)
+  }, [])
+
     // Payload data and url to upload files
     const getUploadParams = ({ meta }) => { return { url: 'https://httpbin.org/post' } }
```

Comment:
I assume this is just a placeholder until the real API is ready

Reply:
fixed changes